src/gwt/org/bbop/apollo/gwt/client/AnnotatorPanel.java

```diff
@@ -428,7 +428,7 @@ private static void updateAnnotationInfo(AnnotationInfo annotationInfo) {
                 geneDetailPanel.updateData(annotationInfo);
                 tabPanel.getTabWidget(1).getParent().setVisible(false);
                 break;
-            case "Transcript":
+            case "transcript":
```

Comment:
this worries me greatly.   What were the repercussions of this?   Everything seemed to work as well the old way.  Was that serendipity?